controllers/articles.controllers.js

```diff
@@ -12,4 +13,17 @@ exports.getArticleById = (req, res, next) => {
         res.status(200).send({ article });
     })
     .catch(next);
-}
\ No newline at end of file
+}
+
+exports.patchArticleById = (req, res, next) => {
+    const { article_id } = req.params;
+    if(Object.keys(req.body).length !== 0){
```

Comment:
This isn't really a robust check, you'd better off letting PSQL deal with an error or at least do some checks in a model.